cms/server/admin/handlers/trainingday.py

```
@@ -0,0 +1,814 @@
+        else:
+            self.redirect(fallback_page)
+
+
+class RemoveTrainingDayHandler(BaseHandler):
+    """Confirm and remove a training day from a training program."""
+
+    @require_permission(BaseHandler.PERMISSION_ALL)
+    def get(self, training_program_id: str, training_day_id: str):
+        training_program = self.safe_get_item(TrainingProgram, training_program_id)
+        training_day = self.safe_get_item(TrainingDay, training_day_id)
+        managing_contest = training_program.managing_contest
```

Ruff (0.14.14)

[error] Local variable `managing_contest` is assigned to but never used

Remove assignment to unused variable `managing_contest`

(F841)